app/src/test/java/com/android/shelflife/model/FoodItemRepositoryFirestoreTest.kt

```diff
@@ -0,0 +1,221 @@
+package com.android.shelflife.model
+
+import android.content.Context
+import com.android.shelfLife.model.foodFacts.FoodFacts
+import com.android.shelfLife.model.foodItem.FoodItem
+import com.android.shelfLife.model.foodItem.FoodItemRepositoryFirestore
+import com.android.shelfLife.model.foodItem.FoodStorageLocation
+import com.google.android.gms.tasks.Tasks
+import com.google.firebase.firestore.*
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import javax.inject.Inject
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.runBlocking
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.*
+import org.robolectric.RobolectricTestRunner
+
+@HiltAndroidTest
+@RunWith(RobolectricTestRunner::class)
+class FoodItemRepositoryFirestoreTest {
+
+  @get:Rule var hiltRule = HiltAndroidRule(this)
+
+  @Inject lateinit var foodItemRepository: FoodItemRepositoryFirestore
```

Comment:
i see we are not injecting some repositories like FoodItemRepositoryFirestore

Reply:
*are

Reply:
Yes to use HILT